generator/src/test/java/com/scottlogic/deg/generator/restrictions/TextualRestrictionsTests.java

```diff
@@ -330,7 +330,7 @@ void createGenerator_withNonContradictingOfLengthAndContainingRegexConstraint_sh
 
     @Test
     void createGenerator_withContradictingOfLengthAndContainingRegexConstraint_shouldCreateNoStrings() {
-        StringRestrictions restrictions = containsRegex("[a-z]{0,9}", false)
+        StringRestrictions restrictions = containsRegex("[a-z]{102}", false)
```

Comment:
why has this changed?

Reply:
the test was erroneous as its containsRegex.  If want to clash with length it need to be a regex with larger length not smaller

Reply:
it only passed as there was a bug (which i am bandaging up in this ticket)